### ISSUE TITLE
Not sure if it's a valid pronunciation

### ISSUE THREAD

**Issue by qiusidong** [cmusphinx/cmudict#8]
As a non-native English speaker, I am not sure about the first pronunciation of the word "natural" in CMU dictionary is valid because the /r/ sound is completely missing. 

> natural N AE1 CH ER0 AH0 L
> natural(2) N AE1 CH R AH0 L

Could anyway tell me whether it's a bug or it's valid? 

Thanks.

**Comment by qiusidong** [comment]
Thanks @nshmyrev !
I looked at a few more examples and found the pronunciations that would omit certain phonemes in the middle and in the end. For example,

> interview IH N ER V Y UW (omit middle /t/)
> interview IH N T ER V Y UW

> invests IH N V EH S <- (omit /ts/)
> invests IH N V EH S S <- (omit /t/)
> invests IH N V EH S T S 

> happening	HH AE P AH N IH NG
> happening	HH AE P N IH NG <- (omit /ɘ/)
> 

Is that because in a fast, continuous speech some phonemes would be naturally omitted?

**Comment by nshmyrev** [comment]
> Is that because in a fast, continuous speech some phonemes would be naturally omitted?

Yes. Overall cmudict design is not that good because cmudict applies reduction rules, but it does not mark them and it also applies reduction rule very inconsistently. pronlex is doing much better here and gives better results too.
